debian/homestead.init.d

```diff
@@ -152,9 +158,7 @@ get_daemon_args()
           diameter_timeout_ms=$(( ($target_latency_us + 499)/500 ))
         fi
 
-        [ "$hss_mar_lowercase_unknown" != "Y" ] || scheme_args="--scheme-unknown=unknown"
```

Comment:
I've basically replaced them with more flexible ones - if you wanted this option, you can now say `hss_mar_scheme_unknown="unknown"` instead.